cgi-bin/LJ/Web.pm

```diff
@@ -886,7 +886,7 @@ sub create_qr_div {
     }
 
     return DW::Template->template_string( 'journal/quickreply.tt', {
-        form_url                => LJ::create_url( '/talkpost_do', host => $LJ::DOMAIN_WEB ),
```

Comment:
Why are we passing `$LJ::IS_SSL` in to many places? Why can't these functions just assume it?

Inside, I would imagine `$ssl = $opts{ssl} // $LJ::IS_SSL;` that way they default to whatever the global variable is, but you can still force set it to 0/1.